src/Process/ProcessBuilder.php

```diff
@@ -61,9 +60,18 @@ public function createArgumentsForCommand($command)
      */
     public function buildProcess(ProcessArgumentsCollection $arguments)
     {
-        $builder = SymfonyProcessBuilder::create($arguments->getValues());
-        $builder->setTimeout($this->config->getProcessTimeout());
-        $process = $builder->getProcess();
+        $commandline = $arguments->getValues();
+
+        // Backwards compatibility layer for Symfony Process < 3.3
+        if (class_exists('Symfony\Component\Process\ProcessBuilder')) {
```

Comment:
Maybe it's better to add a `generateCliCommand(): string` or `createProcess(): Process` method to the `ProcessArgumentsCollection` so that it is reusable?

Reply:
Personally I'd prefer `generateCliCommand(): string` approach (in order to not couple the collection too much to Symfony). On the other hand, it will contain this "backwards compatibility layer" anyway, and I feel it might be a bit _easier_ to just have a `createProcess(): Process`